developer/themes/templates.md

```diff
@@ -1,3 +1,10 @@
+
+---
+**NOTE:**
+This page's content is deprecated and no longer relevant. Reaction is in the progress of completely eliminating all usages of Blaze technology and replacing it through React components. Prolonged usage of Blaze templates will rule an upgrade path to newer Reaction versions out.
```

Comment:
I think you probably meant "process" rather than "progress".

Also one suggestion. Take it or leave it.
A less awkward less sentence might be:

"Continuing to use Blaze templates may make upgrading to new versions of Reaction more difficult."